src/components/NcRelatedResourcesPanel/NcTeamResources.vue

```diff
@@ -0,0 +1,293 @@
+<template>
+	<div v-if="appEnabled && isVisible" class="related-resources__team">
+		<div class="related-resources__header">
+			<h5>{{ t('Related team resources') }}</h5>
+		</div>
+		<div v-for="team in teamResources"
+			:key="team.teamId"
+			class="related-team"
+			:class="{ 'related-team__open': open(team.teamId) }">
+			<h5 class="related-team__header">
+				<a class="related-team__link" @click="toggleOpen(team.teamId)">
+					<AccountGroup :size="20" />
+					{{ team.displayName }}
+				</a>
+				<NcActions>
+					<NcActionLink :href="team.link" :label="t('View team')">
+						<template #icon>
+							<OpenInNew :size="20" />
+						</template>
+					</NcActionLink>
+				</NcActions>
+				<NcButton type="tertiary" @click.stop="toggleOpen(team.teamId)">
+					<ChevronUp v-if="open(team.teamId)"
+						:size="20" />
+					<ChevronDown v-else
+						:size="20" />
+				</NcButton>
+			</h5>
+
+			<div v-if="open(team.teamId)">
+				<div v-for="provider in teamProviders(team.teamId)"
+					:key="provider.id"
+					class="related-team-provider">
+					<h6 v-if="provider.resources.length > 0">
+						{{ provider.name }}
+					</h6>
+					<ul>
+						<li v-for="resource in provider.resources" :key="resource.url" class="related-team-resource">
+							<a :href="resource.url" class="related-team-resource__link">
+								<span v-if="resource.iconEmoji" class="resource__icon">
+									{{ resource.iconEmoji }}
+								</span>
+								<NcIconSvgWrapper v-else-if="resource.iconSvg"
+									class="resource__icon"
+									:svg="resource.iconSvg"
+									:size="20" />
+								<span v-if="resource.iconUrl" class="resource__icon">
```

Comment:
I'm not exactly sure I get the logic here: If `resource.iconEmoji` is set, it will be displayed, otherwise `resource.iconSvg` will be displayed if set. But `resource.iconUrl` will always be displayed. Shouldn't the latter be a `v-else-if` as well?

Reply:
Yes, that should indeed be a v-else-if, fixed